source/optional_features/digital_wallets/index.html.md.erb

```diff
@@ -0,0 +1,64 @@
+---
+title: Digital wallets
+weight: 136
+---
+
+# Digital wallets
+
+You can enable [Google Pay](https://pay.google.com) and [Apple Pay](https://www.apple.com/uk/apple-pay/) [external links] to take payments from your users.
```

Comment:
If you’re not logged in to a Google account, https://pay.google.com/ takes you to a login page. I think a URL like https://pay.google.com/intl/en_uk/about/ would be better (this does not require you to be logged in to Google).

Reply:
will change